examples/greenfield/storageclass.yaml

```diff
@@ -12,7 +12,23 @@ parameters:
   secretNamespace: s3-provisioner
   s3Endpoint: http://s3-reg-1.landemo1.cloudian.eu
   iamEndpoint: http://iam.landemo1.cloudian.eu:16080
+
   # Set storagePolicyId to create buckets with specified policy
   #storagePolicyId: <policy id>
+
+  # Provide an IAM policy document to override the default IAM policy
+  # of read+write access to the bucket.
+  # Omit the "Resource" field - it will be set to only allow access to the claimed bucket
+  # For example to set a bucket read-only, uncomment the following:
+  #iamPolicy: |
+  #  {
+  #    "Version": "2012-10-17",
+  #    "Statement": [{
+  #            "Sid": "AllowAll",
+  #            "Effect": "Allow",
+  #            "Action": ["s3:HeadObject", "s3:ListBucket", "s3:GetObject"]
+  #    }]
+  #  }
+
```

Comment:
This example is counter-intuitive for greenfield!  You'll just get an empty bucket that you can see is empty and no more!

Reply:
I'll change it to write only

Reply:
Does that make sense either?  A greenfield bucket is not accessibly by anyone else...

Reply:
Ok, I'll remove it from greenfield entirely then :-p

Reply:
Makes sense - am struggling to see why a greenfield bucket wouldn't just have full access.